plugins/content/local/writer.go

```diff
@@ -137,6 +138,20 @@ func (w *writer) Commit(ctx context.Context, size int64, expected digest.Digest,
 		return err
 	}
 
+	// Enable content blob integrity verification if supported
+
+	var (
+		integritySupported bool
+		supportErr         error
+	)
+	if integritySupported, supportErr = fsverity.IsSupported(w.s.root); integritySupported {
```

Comment:
I think the `IsSupported` cache should be handled in content local plugin.
The `fsverity.IsSupported` is just handled once. If we change the input from A to B, the cache might not be correct.

Reply:
I've updated the implementation to cache `IsSupported` value in the content plugin.